apps/client/src/app/components/user-account-access/user-account-access.html

```diff
@@ -10,8 +10,19 @@
   </h1>
   <gf-access-table
     [accesses]="accesses"
-    [hasPermissionToCreateAccess]="hasPermissionToCreateAccess"
     [showActions]="hasPermissionToDeleteAccess"
     (accessDeleted)="onDeleteAccess($event)"
   ></gf-access-table>
+  <div *ngIf="hasPermissionToCreateAccess" class="fab-container">
+    <a
+      class="align-items-center d-flex justify-content-center"
+      color="primary"
+      i18n
```

Comment:
As there is no button label, you can remove the i18n directive 🙂

Reply:
done